### ISSUE TITLE
WCAG 1.4.3: Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (h4)

### ISSUE THREAD

**Issue by Tjoosten** [vl-wbk/vl-woordenboek#199]
#### Issue
`Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[VL. woordenboek | Gerief](http://vlaams-woordenboek.test/willekeurig-woordenboek-artikel)
#### Element path
h4
#### Snippet
`<h4 class="text-gold float-start">Gerief</h4>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 2.79 (foreground color: #c58c00, background color: #f9f9f9, font size: 16.2pt (21.6px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 136.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.46.0 (axe-core 4.10.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
